sympy/solvers/bivariate.py

```diff
@@ -208,8 +214,16 @@ def _solve_lambert(f, symbol, gens):
     if lhs.is_Mul:
         lhs = expand_log(log(lhs))
         rhs = log(rhs)
+        if lhs.is_Add and lhs.has(t):
+            llhs1, llhs2 = map(lambda i: lhs.xreplace({t: i}), (symbol, -symbol))
+            sol1, sol2 = map(lambda i: _solve_lambert(i - rhs, symbol, gens), (llhs1, llhs2))
+            return list((set(sol1 + sol2)))
+
+    if lhs.has(t):
```

Comment:
If it has `t`, means it has even powers which implies the equation should diverge to two: with `+symbol` and `-symbol`.
Why it's not done here?

Reply:
Not every even_degree is leading to two equations. Only those equation in which even degree is affecting the cofficients in a*log(b*x + c) are branching out to two equations which is being taken careof by applying `expand_log` above.